R/utils.R

```diff
@@ -143,7 +143,9 @@ merge_chapters = function(files, to, before = NULL, after = NULL, orig = files)
   preview = opts$get('preview'); input = opts$get('input_rmd')
   content = unlist(mapply(files, orig, SIMPLIFY = FALSE, FUN = function(f, o) {
     x = readUTF8(f)
-    x = if (preview && !(o %in% input)) create_placeholder(x) else {
+    x = if (preview && !(o %in% input)) {
+      create_placeholder(x)
+    } else {
```

Comment:
In the future, please refrain from mixing cosmetic changes with essential changes in the same pull request.